.github/label-commenter-config.yml

```diff
@@ -23,6 +23,10 @@ labels:
               <p>
                 If the issue is valid but there is no time to troubleshoot it, consider adding the <code>help wanted</code> label.
               </p>
+              <p>
+                If the issue requires changes or fixes from an external project (e.g., chromedriver, geckodriver, w3c),
+          add the applicable <code>G-*</code> label, and it will provide the correct link and auto-close the issue.
+              </p>
```

Comment:
```suggestion
              <p>
                If the issue requires changes or fixes from an external project (e.g., ChromeDriver, GeckoDriver, W3C),
               add the applicable <code>G-*</code> label, and it will provide the correct link and auto-close the 
               issue.
              </p>
```